docs/src/pages/docs/crocks/Maybe.md

```diff
@@ -990,6 +990,65 @@ find(isEven, [ 1, 2, 3, 4, 5 ])
 //=> Just 2
 ```
 
+#### getPath
+
+`crocks/Maybe/getPath`
+
+```haskell
+getPath :: [ (String | Integer) ] -> a -> Maybe b
+```
+
+While [`getProp`](#getprop) is good for simple, single-level structures, there
+may come a time when you have to work with nested POJOs or Arrays. When you run
+into this situation, just pull in `getPath` (formally known as `propPath`) and
+pass it a left-to-right traversal path of keys, indices or a combination of
+both. This will kick you back a function that behaves just
```

Comment:
```suggestion
both as an array. This will kick you back a function that behaves just
```

unless we plan on supporting any foldable, why not be explicit?